lib/parallel_tests.rb

```diff
@@ -17,7 +18,7 @@ def determine_number_of_processes(count)
       [
         count,
         ENV["PARALLEL_TEST_PROCESSORS"],
-        Parallel.processor_count
```

Comment:
need to remove that method then ?
... best make it call Concurrent.available_processor_count and print a deprecation

Reply:
Is it OK to remove `Parallel.processor_count`? I thought we can't remove it because it consider env(`PARALLEL_PROCESSOR_COUNT`), but `Concurrent.available_processor_count` not. 

Is it better to use `Concurrent.available_processor_count` inside `Parallel.processor_count` or add a new method like `Parallel.available_processor_count`?

Reply:
it would be awesome to have Parallel.processor_count use Concurrent.available_processor_count
because otherwise the copy-pasting just continues and afaik these methods are supposed to do the same thing

Reply:
possibly even make `Parallel.processor_count` show a deprecation and tell users to use Concurrent.available_processor_count

Reply:
Thanks. I've created the PR to `parallel`.
https://github.com/grosser/parallel/pull/348